core/cli/commands/smell.py

```
@@ -0,0 +1,578 @@
+                    smell_vec = embedder.embed_texts([smell_name], task="retrieval.query")[0].tolist()
+
+                    similarity = _cosine_similarity(file_vec, smell_vec)
+                    probe_results.append({
+                        "cs_id": cs_id,
+                        "smell_name": smell_name,
+                        "file": file_str,
+                        "cosine_similarity": round(similarity, 4),
+                        "pass": similarity >= 0.5,
+                    })
+
+                except Exception as probe_err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)